src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -471,13 +483,16 @@ const ModalCardDetails = ({
             <CheckboxContainer>
               <Checkbox
                 value="checkedA"
-                inputProps={{ 'aria-label': 'Checkbox A' }}
+                inputProps={{ 'aria-label': 'Terms and Conditions' }}
```

Comment:
were these two left as "checkbox a" and "b" on purpose? or just a lack of a better name for them

Reply:
This looks like something pretty old that we just never changed. Looks like the last person to touch these was @wilsonj806 and they were made by @ath-chen, either of you have any input? While we're changing the aria-labels, should we change the values to something more meaningful too?

Reply:
i think these were just never updated haha, yeah the value should be changed as well. thanks for catching and updating!

Reply:
i'm not sure what's using this down stream, but i changed the label for now. we can (and should) create a ticket to address the value. i did a search-all for "checkedA" couldn't find any usages. unless you can tell me @ath-chen that it's not "used" anywhere yet. then i can updated in this PR with minimal scope creep